### PR TITLE
New docs structure

### PR DESCRIPTION
**Pull request by iAdramelk**
!!! Need to be merged at the same time as the main docs repo.

Updated docs structure:

```
docs/
  VERSION/
    img/
    pages/
      *.md
    config.json
```

Changes:

- Config is now placed inside the version folder.
- *.md files moved to subfolder `pages/` so we can symlink it to the `pages/` directory of the docs site without mixing it with images, READMEs and other assets.
- Paths to images inside the *.md files are updated to include additional parent folder.

Some caveats:

- I left `milv.config.yaml` files inside the docs folders, but they will probably not work anymore.
- Folder for `3.1` had readme file that is not part of the docs. Not sure if it used anywhere, left in the folder root for now.
- Versions `3.1`-`4.1` have `draw.io` folders with sources of some images. Left them there for now, but we can remove them if we don't need them anymore.
- For now build scripts and theme from original docs in the folder, but we probably need to remove them someday.

### CODE REVIEW

**Review by klizhentas — APPROVED** [review]

File: docs/4.3/pages/aws-oss-guide.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ stores the state of the system and S3 will store audit logs.
 ![AWS Intro Image](../img/aws/aws-intro.png)
 
 ### EC2 / Autoscale
-To run Teleport in a HA configuration we recommend using m4.large instances. It's best practice to separate the proxy and authentication server, using autoscaling groups for both machines. We have pre-built AMIs for both Teleport OSS and Enterprise editions. Instructions on using these [AMIs are below](#single-oss-teleport-amis-manual-gui-setup).
+To run Teleport in a HA configuration we recommend using m4.large instances. It's best practice to separate the proxy and authentication server, using autoscaling groups for both machines. We have pre-built AMIs for both Teleport OSS and Enterprise editions.
```
Comment (iAdramelk) [inline docs/4.3/pages/aws-oss-guide.md:70]: This section was removed in 4.3, so I remove reference for it. If it was moved elsewhere, then we should add correct link instead.

File: docs/4.4/pages/aws-oss-guide.md [hunk 2]
```diff
@@ -67,7 +67,7 @@ stores the state of the system and S3 will store audit logs.
 ![AWS Intro Image](../img/aws/aws-intro.png)
 
 ### EC2 / Autoscale
-To run Teleport in a HA configuration we recommend using m4.large instances. It's best practice to separate the proxy and authentication server, using autoscaling groups for both machines. We have pre-built AMIs for both Teleport OSS and Enterprise editions. Instructions on using these [AMIs are below](#single-oss-teleport-amis-manual-gui-setup).
+To run Teleport in a HA configuration we recommend using m4.large instances. It's best practice to separate the proxy and authentication server, using autoscaling groups for both machines. We have pre-built AMIs for both Teleport OSS and Enterprise editions.
```
Comment (iAdramelk) [inline docs/4.4/pages/aws-oss-guide.md:70]: Same as above. This section was removed in 4.3.

File: docs/5.0/pages/aws-oss-guide.md [hunk 3]
```diff
@@ -67,7 +67,7 @@ stores the state of the system and S3 will store audit logs.
 ![AWS Intro Image](../img/aws/aws-intro.png)
 
 ### EC2 / Autoscale
-To run Teleport in a HA configuration we recommend using m4.large instances. It's best practice to separate the proxy and authentication server, using autoscaling groups for both machines. We have pre-built AMIs for both Teleport OSS and Enterprise editions. Instructions on using these [AMIs are below](#single-oss-teleport-amis-manual-gui-setup).
+To run Teleport in a HA configuration we recommend using m4.large instances. It's best practice to separate the proxy and authentication server, using autoscaling groups for both machines. We have pre-built AMIs for both Teleport OSS and Enterprise editions.
```
Comment (iAdramelk) [inline docs/5.0/pages/aws-oss-guide.md:70]: Same as above.

File: docs/5.0/pages/enterprise/ssh-kubernetes-fedramp.md [hunk 4]
```diff
@@ -20,8 +20,8 @@ government agencies.
...
+| [AC-20 Use of External Information Systems](https://nvd.nist.gov/800-53/Rev4/control/AC-20)  | Teleport supports connecting multiple independent clusters using a feature called [Trusted Clusters](../trustedclusters/). When allowing access from one cluster to another, roles are mapped according to a pre-defined relationship of the scope of access.|
+| [AU-03 Audit and Accountability](https://nvd.nist.gov/800-53/Rev4/control/AU-3) – Content of Audit Records and [AU-12 Audit Generation](https://nvd.nist.gov/800-53/Rev4/control/AU-12) | Teleport contains an [Audit Log](../architecture/authentication/#audit-log) that records cluster-wide events such as: <br />• Failed login attempts.<br />• Commands that were executed (SSH “exec” commands).<br /> • Ports that were forwarded. <br />• File transfers that were initiated.|
```
Comment (iAdramelk) [inline docs/5.0/pages/enterprise/ssh-kubernetes-fedramp.md:24]: These link didn't work according to the latest semrush report, so I fixed them. Not sure how they worked before.

**Review by iAdramelk — COMMENTED** [review]
@sasha Please check comments.

Also there is a broken link in https://github.com/gravitational/teleport/blob/master/docs/4.4/preview/teleport-database-access.md#L222 (link to https://github.com/gravitational/teleport/blob/roman/rfd/dba/rfd/0011-database-access.md), not sure what to do with it.

**Pull request merged**